CHANGELOG.md

```diff
@@ -14,6 +14,13 @@ and this module adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.
 
 ### Changed
 
+## [4.0.0]
+
+### Changed
+
+- changed parameter type from to string since
+- changed deschription
```

Comment:
```suggestion
- changed type of parameter `virtualNetworkResourceId` from array to string, because multiple Ids weren't working and used anyway. Use multiple policy assignments instead.
```